src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/EnumConverterTests.cs

```diff
@@ -186,8 +186,8 @@ public void ConvertFileAttributes(bool useGenericVariant)
                 options);
             Assert.Equal(@"""directory, compressed, integrity_stream""", json);
 
-            json = JsonSerializer.Serialize(FileAttributes.Compressed & FileAttributes.Device, options);
-            Assert.Equal(@"0", json);
+            json = JsonSerializer.Serialize((FileAttributes)(-1), options);
+            Assert.Equal(@"-1", json);
```

Comment:
I think the previous test was validating that an unknown value is serialized as a number.

`FileAttributes.Compressed & FileAttributes.Device` is now `None`. So, the test fails on net462 but succeeds on net8. I replaced the value with an unknown value (-1).